app/controllers/staff/activity_forms_controller.rb

```diff
@@ -105,4 +110,9 @@ def update_wizard_status
       @activity.wizard_status = step
     end
   end
+
+  def reset_sector_answer
+    @activity.update(sector: nil, wizard_status: "editing")
```

Comment:
I'm cautious about adding a new wizard_status for editing. I think it introduces a new scenario where a user marks an activity as editing but then fails to complete their transaction (for whatever reason) which marks it back as complete. This could cause odd behaviour. It opens questions like: what happens if 2 people edit the same document at the same time?

Could we rely on this editing state to live only in a query parameter for the request? We're already headed that way with `editing_until`. Perhaps `editing=true` and `editing_step=sector_category`?

Reply:
Another idea: A new field that tracks the state of when an activity has been fully filled out. If that's marked as true, we can infer that all future updates are edits. We'd still need to pass the step around in a query param.